site.properties

```diff
@@ -129,17 +125,28 @@ ldap.lastNameAttribute  = sn
 #liquibase.jdbc.username = apromore
 #liquibase.jdbc.password = MAcri
 
+impala.jdbc.driver   = com.cloudera.impala.jdbc41.Driver
+impala.jdbc.url      = ${env.IMPALA_LINK}
+impala.jdbc.username = apromore
+impala.jdbc.password = MAcri
+impala.etl.dataStore = ${env.DATA_STORE}
```

Comment:
I believe we're trying to avoid committing ETL-related config into site.properties since it's part of the secret ApromoreEE stuff.

Reply:
Revereted to blank values, per discussion with Simon [DONE].